src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -159,7 +204,24 @@ export const WorkspaceDashboard = _.flow(
             ])
           ],
           [!!description, () => h(Markdown, [description])],
-          () => div({ style: { fontStyle: 'italic' } }, ['No description added']))
+          () => div({ style: { fontStyle: 'italic' } }, ['No description added'])),
+        _.some(_.startsWith('library:'), _.keys(attributes)) && h(Fragment, [
+          div({ style: styles.header }, ['Dataset Attributes']),
+          h(SimpleTable, {
+            rows: _.flow(
```

Comment:
nit: something about that first top line makes me think that there should be bold labels for the columns or something. It feels a bit like an unlabeled table

Reply:
Technically it is just a table without headers. But given that the headers would just be something like 'attribute' and 'value', they seemed unnecessary. I'm inclined to leave it for now, pending a full design pass.